src/Sylius/Bundle/CurrencyBundle/Form/Type/CurrencyType.php

```diff
@@ -30,6 +30,12 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ->add('exchangeRate', 'number', [
                 'label' => 'sylius.form.currency.exchange_rate',
             ])
+            ->add('enabled', 'checkbox', [
+                'label' => 'sylius.form.currency.enabled',
+            ])
+            ->add('base', 'checkbox', [
+                'label' => 'sylius.form.currency.base',
+            ])
```

Comment:
This should not be editable anywhere, only created during install and cannot be changed from UI/API. :) Just remove this field and it will be fine!